package.json

```diff
@@ -2,18 +2,136 @@
   "name": "wp-dev-lib",
   "version": "1.0.0",
   "description": "Common code used during development of WordPress plugins and themes",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "repository": "xwp/wp-dev-lib",
+  "engines": {
+    "node": ">=6.11.x"
+  },
   "keywords": [
     "wordpress",
     "plugins",
     "themes",
     "development",
     "tools"
   ],
-  "author": "Weston Ruter <weston@xwp.co> (https://xwp.co)",
+  "author": {
+    "name": "Weston Ruter",
+    "email": "weston.ruter@xwp.co",
+    "url": "https://www.xwp.co"
+  },
+  "contributors": [
+    {
+      "name": "Piotr Delawski",
+      "email": "piotr.delawski@xwp.co"
+    },
+    {
+      "name": "Mike Crantea",
+      "email": "mike.crantea@xwp.co"
+    },
+    {
+      "name": "Justin Kopepasah",
+      "email": "justin.kopepasah@xwp.co",
+      "url": "https://kopepasah.com"
+    }
+  ],
   "license": "MIT",
-  "homepage": "https://github.com/xwp/wp-dev-lib#readme"
+  "homepage": "https://github.com/xwp/wp-dev-lib#readme",
+  "scripts": {
+    "build:self": "./build-workflows.sh",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "browserslist": [
+    "last 2 Chrome versions",
+    "last 2 Firefox versions",
+    "last 2 Safari versions",
+    "last 2 Edge versions",
+    "last 2 Opera versions",
+    "last 2 iOS versions",
+    "last 1 Android version",
+    "last 1 ChromeAndroid version",
+    "ie 11",
+    "> 1%"
+  ],
+  "babel": {
+    "presets": [
+      [
+        "env"
+      ]
+    ]
+  },
+  "stylelint": {
+    "extends": "stylelint-config-wordpress/scss",
+    "plugins": [
+      "stylelint-no-unsupported-browser-features"
+    ],
+    "rules": {
+      "string-quotes": "single",
+      "color-hex-length": null,
+      "function-parentheses-space-inside": null,
+      "function-url-quotes": "always",
+      "at-rule-empty-line-before": [
+        "always",
+        {
+          "except": [
+            "first-nested",
+            "blockless-after-same-name-blockless"
+          ],
+          "ignore": [
+            "after-comment"
+          ],
+          "ignoreAtRules": [
+            "else",
+            "elseif"
+          ]
+        }
+      ],
+      "plugin/no-unsupported-browser-features": [
+        true,
+        {
+          "severity": "warning"
+        }
+      ],
+      "selector-pseudo-element-colon-notation": "double",
+      "value-keyword-case": null
```

Comment:
Thanks for the detailed explanation @delawski, I now see what I was missing here was a bit of context that @westonruter has now clarified. As is obvious that this repo is a XWP project, I also know quite a few people outside of XWP use this repo(or least parts of it) so diverging from WordPress' Coding Standards had me a little perplexed.

Going the route of creating `stylelint-config-xwp` would be 👌 

Your guess in regard to `selector-pseudo-element-colon-notation` is correct, it was because of IE8, I've created https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/issues/165 to update to double colons 👍